utils/data.py

```diff
@@ -441,8 +497,15 @@ def fload_and_patch(self, selectedPatients, selectedDatasets):
                                                                                           self.patchOverlap,
                                                                                           labelMask_ndarray, 0.5,
                                                                                           dataset)
+                                dPatchesOfMask_phase, dLabelsMask_phase = fRigidPatching_maskLabeling(labelMask_ndarray_phase,
```

Comment:
this is not required!
The segmentation mask stays the same for both magnitude and phase